src/webgldrawer.js

```diff
@@ -870,9 +865,14 @@
             let position;
 
             let data = cache.data;
+            let canvas = false;
```

Comment:
It would be better to call this `isCanvas` or such. Otherwise, just looking at the name, you would think it was a canvas, not a flag.